doc/api/cli.md

```diff
@@ -144,6 +144,9 @@ added: v12.12.0
 
 Enable experimental Source Map V3 support for stack traces.
 
+Note, that `--enable-source-maps` does not work in conjunction with the
+`Error.prepareStackTrace` hook.
```

Comment:
Since we have not decided whether this will be permant I think it's worth adding something like "currently" or "at the moment" here so people know it's not hopeless :)